src/Compilers/Core/Portable/StrongName/StrongNameFileSystem.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.Diagnostics;
+using System.IO;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis
+{
+    internal class StrongNameFileSystem
+    {
+        internal readonly static StrongNameFileSystem s_StrongNameFileSystemInstance = new StrongNameFileSystem();
```

Comment:
`internal` fields should start with uppercase letter. Consider renaming to simply `Instance`.

Reply:
Done.